apps.yaml

```diff
@@ -396,3 +396,19 @@ EVA:
     categories:
         - data-visualization
         - data-analysis
+
+SEI-Active learning workflow:
```

Comment:
please remove the spaces.

Reply:
Sorry, I was not clear before. I meant the spaces in the name, so something like `SEI-Active_learning_workflow`